servers.json

```diff
@@ -1269,6 +1269,20 @@
         "name": "Sharkmc Network"
         "addresses": [
             "sharkmc.es"
```

Comment:
You are missing a `]` for closing the `addresses` array

Reply:
I already made the change sorry